src/Network/NWProtocolOptions.cs

```diff
@@ -18,85 +18,86 @@
 
 namespace Network {
 
-	[Watch (6,0), TV (13,0), Mac (10,15), iOS (13,0)]
-	public enum NWIPLocalAddressPreference {
-		Default = 0,
-		Temporary = 1,
-		Stable = 2,
-	}
-
 	[TV (12,0), Mac (10,14), iOS (12,0)]
 	[Watch (6,0)]
 	public class NWProtocolOptions : NativeObject {
 		public NWProtocolOptions (IntPtr handle, bool owns) : base (handle, owns) {}
 
 		[DllImport (Constants.NetworkLibrary)]
-		static extern OS_nw_protocol_definition nw_protocol_options_copy_definition (IntPtr options);
+		protected static extern OS_nw_protocol_definition nw_protocol_options_copy_definition (IntPtr options);
```

Comment:
Can these P/Invokes be internal instead of protected? That way they're not part of our public API.